internal/api/resolver_mutation_performer.go

```diff
@@ -504,3 +532,93 @@ func (r *mutationResolver) PerformersDestroy(ctx context.Context, performerIDs [
 
 	return true, nil
 }
+
+func (r *mutationResolver) PerformerMerge(ctx context.Context, input PerformerMergeInput) (*models.Performer, error) {
+	srcIDs, err := stringslice.StringSliceToIntSlice(input.Source)
+	if err != nil {
+		return nil, fmt.Errorf("converting source ids: %w", err)
+	}
+
+	// ensure source ids are unique
+	srcIDs = sliceutil.AppendUniques(nil, srcIDs)
+
+	destID, err := strconv.Atoi(input.Destination)
+	if err != nil {
+		return nil, fmt.Errorf("converting destination id: %w", err)
+	}
+
+	// ensure destination is not in source list
+	if slices.Contains(srcIDs, destID) {
+		return nil, errors.New("destination scene cannot be in source list")
+	}
+
+	var values *models.PerformerPartial
+	var imageData []byte
+	var legacyURLs *LegacyURLs
```

Comment:
We're only using this to validate that its not used, so this can be declared inside the `if` statement.